kafka-payload-processor-shaded-tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaPayloadProcessorTest.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.streamnative.pulsar.handlers.kop;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.pulsar.client.api.Schema;
+import org.apache.pulsar.client.impl.MessageIdImpl;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class KafkaPayloadProcessorTest {
+
+    @Test
+    public void testValues() throws Exception {
```

Comment:
Can we add some tests to verify bypassing the control batch?

Reply:
This PR only tests the `kafka-payload-processor` dependency could work with `pulsar-client` dependency. So this test is more like a smoke test. I think we can use other PRs to add more tests, including tests for keys and headers.